ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -299,13 +299,31 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         }
     }
 
+    /// Deprecated:
     /// Changes the `isHidden` state with a completion handler.
     /// - Parameters:
     ///   - isHidden: The new value.
     ///   - animated: Indicates if the change should be animated. The default value is `true`.
     ///   - completion: Closure to be called when the state change completes.
+    @available(*, deprecated, message: "Use setHidden:unhiddenTargetState:animated:completion:")
     @objc public func setIsHidden(_ isHidden: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
-        let targetState: BottomSheetExpansionState = isHidden ? .hidden : .collapsed
+        setHidden(isHidden, animated: animated, completion: completion)
+    }
+
+    /// Changes the `isHidden` state with a completion handler.
+    /// - Parameters:
+    ///   - isHidden: The new value.
+    ///   - unhiddenTargetState: The state the sheet should expand to when not hidden.
```

Comment:
Related to other comments, but a bit odd from an API perspective that someone setting this to hidden can also specify an unhiddenTargetState. Feels like it should maybe have separate show/hide (or present/dismiss) APIs? Which also better matches up with usual UIKit semantics?

And that can of course call into setHidden internally to avoid duplicating logic?

Reply:
Or maybe easiest would be to just expose setExpansionState directly?